tests/test_runner/test_runner.py

```diff
@@ -38,22 +38,48 @@ def test_epoch_based_runner():
         def batch_processor():
             pass
 
-        _ = EpochBasedRunner(model, batch_processor)
+        _ = EpochBasedRunner(
+            model, batch_processor, logger=logging.getLogger())
 
     with pytest.raises(TypeError):
         # batch_processor must be callable
-        model = OldStyleModel()
-        _ = EpochBasedRunner(model, batch_processor=0)
```

Comment:
`OldStyleModel()` should be used. `batch_processor` shouldn't co-exist with `train_step`.